### DISCUSSION TITLE
Add option to customise tabs rounding radius

### DISCUSSION THREAD

**Discussion by hoshi22** [black7375/Firefox-UI-Fix#967] [Ideas]
Hi guys, would be great to add an option to customize tabs rounding radius. For me tabs looks only slightly rounded, I would like to have an option to set a bigger rounding radius. I guess it would be nice option to everyone.

**Comment by black7375** [comment]
Do you want it to have some roundness?
Check it out and if it's too trivial, I'll walk you through how you can modify your `userChrome.css`.
